docs/networks/testnet/running-a-node.md

```diff
@@ -137,13 +72,5 @@ Ask your question in the validators channel on the [official LUKSO Discord serve
 
 ## Further Reads
 
-For extended help or advise for setting up your node beyond the LUKSO CLI, you can check out the following links.
```

Comment:
why are we removing the community links?

Reply:
They are already in the main page:

https://docs.lukso.tech/networks/mainnet/running-a-node